### PR TITLE
Demos: Issue 635

### PR DESCRIPTION
**Pull request by xuqiancheng**
Check all jpscore demo files.
- Demo 6_ship is removed.

- Other demos has been renamed.

- All the demos work well now.

- A README.md is created for each demo.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -187,4 +187,6 @@ js
 .DS_Store
 JuPedSim-blx.bib
 _tex/
-titlepage.tex
\ No newline at end of file
+titlepage.tex
+
+/scripts/deps/
```
Comment (zeroset) [inline .gitignore]: Please remove this ignore. It is user specific where you create the dependencies folder.

File: docs/pages/jpscore/jpscore_demos.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+sidebar: jupedsim_sidebar
+folder: jpscore
+permalink: jpscore_demos.html
+summary: Introduction of demos in jpscore.
+
+last_updated: Feb 04, 2019
+---
+
+## 01_corridor
+### Description
```
Comment (chraibi) [inline docs/pages/jpscore/jpscore_demos.md]: Please change structure of the file, for example as follows:
## 01_corridor 

**Description**

**Settings**

File: docs/pages/jpscore/jpscore_demos.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+This demo shows 20 agents moving from left to right in a rectangular corridor.
+
+The length of the corridor is 10 meters and the width is 2 meters.
+
+### Settings
+- Model: [Generalized Centrifugal Force Model](http://www.jupedsim.org/jpscore_operativ.html#generalized-centrifugal-force-model)
```
Comment (chraibi) [inline docs/pages/jpscore/jpscore_demos.md]: Please don't use these links. 

You can link to internal files by just using the names of the files.

For example 

```
[Generalized Centrifugal Force Model](jpscore_operativ.html#generalized-centrifugal-force-model)
```

File: docs/pages/jpscore/jpscore_demos.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+- Router: [Global shortest path](http://www.jupedsim.org/jpscore_routing.html#global-shortest-path)
+- Strategy: [Strategy 2](http://www.jupedsim.org/jpscore_direction.html#strategy-2)
+
+### Screenshot
```
Comment (chraibi) [inline docs/pages/jpscore/jpscore_demos.md]: remove this title
(.. and everywhere)

File: docs/pages/jpscore/jpscore_demos.md (hunk 2 above)
Comment (xuqiancheng) [reply docs/pages/jpscore/jpscore_demos.md]: > Please change structure of the file, for example as follows:

Thanks, done

**Pull request merged**